Velocity_Model/scripts/get_z.py

```diff
@@ -0,0 +1,31 @@
+import argparse
+from pathlib import Path
+import os
+
+from qcore import formats
+from Velocity_Model.z import extract_z
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-ll", help="Station ll file", required=True)
+    parser.add_argument(
+        "-z",
+        "--z-type",
+        choices=[
+            "Z1.0",
+            "Z2.5",
+        ],  # , "vs30", "vs500"], -- couldn't get vs30 or vs500 working despite documentation saying it should work
+        help="Selects what Z type to extract: [%(choices)s]",
+        nargs="+",
+        required=True,
+    )
```

Comment:
Why are these required flagged arguments, rather than standard positional arguments?

Reply:
I prefer flagged arguments over positional so it's easy to remember the order / it doesn't matter.